apiserver/pkg/server/job_server.go

```diff
@@ -80,14 +80,16 @@ func (s *RayJobServer) ListRayJobs(ctx context.Context, request *api.ListRayJobs
 }
 
 // Finds all Jobs in all namespaces.
-func (s *RayJobServer) ListAllRayJobs(ctx context.Context, _ *api.ListAllRayJobsRequest) (*api.ListAllRayJobsResponse, error) {
-	jobs, err := s.resourceManager.ListAllJobs(ctx)
+func (s *RayJobServer) ListAllRayJobs(ctx context.Context, request *api.ListAllRayJobsRequest) (*api.ListAllRayJobsResponse, error) {
+	// Leave the namespace empty to list all jobs in all namespaces.
+	jobs, continueToken, err := s.resourceManager.ListJobs(ctx, "" /*namespace*/, request.Continue, request.Limit)
```

Comment:
This seems to be inconsistent with K8s behavior. That is, `kubectl get pods` lists all Pods in the default namespace.

Reply:
TLDR: http endpoints are different.

Check with command `k get jobs -v=8`, the HTTP action and endpoint we request is `GET https://34.19.12.93/apis/batch/v1/namespaces/default/jobs?limit=500`

For kuberay apiserver, we don't fill in namespace field, and documentation specifies we want all jobs under all namespaces.
https://github.com/ray-project/kuberay/blob/06367a34b17eeecee6bee51cf4ca7b24ac3412e2/proto/job.proto#L51-L56

Reply:
Okay, if it's already documented, it's fine to retain the current behavior.